sample-applications/blazing-coffee/BlazingCoffee/Client/wwwroot/js/themeChooser.js

```diff
@@ -1,30 +1,50 @@
 ﻿window.themeChooser = {
     themeVersion: "3.5.0",
+    themeList: [
+        { themeValue: "default-ocean-blue", mainTheme:"default", isSwatch: true },
+        { themeValue: "default", mainTheme: "default", isSwatch: false },
+        { themeValue: "bootstrap", mainTheme: "bootrstrap", isSwatch: false },
+        { themeValue: "material", mainTheme: "material", isSwatch: false },
+    ],
+    defaultTheme: { themeValue: "default-ocean-blue", mainTheme: "default", isSwatch: true },
     getTheme: function () {
-        return window.localStorage['ThemeSetting'];
+        let themeSetting = JSON.parse(window.localStorage['ThemeSetting'] || "{}");
```

Comment:
Something like:
```js
let themeSetting = JSON.parse(window.localStorage.get('ThemeSetting') || {};
```
looks to me as a more standard way of writing this - there is not need to parse "{}" as we know the outcome anyway.